.changes/next-release/feature-788a9810-8e83-4f9a-8b04-204c6b3e372d.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Support timeout and memory size settings in run configuration"
```

Comment:
this looks strange ....

Reply:
I think this is due to the squash merge that is in the history of this branch